tests/System/ImportTest.php

```diff
@@ -176,7 +176,7 @@ public function getApiTestsToRun()
                 ],
             ]],
 
-            ['MarketingCampaignsReporting', [
+            ['MarketingCampaignsReporting.getKeyword', [
```

Comment:
Thank you for your help figuring this out @AltamashShaikh . Turns out I was looking at the wrong build when I checked if it worked.

Reply:
@snake14 we can revert it to old code

Reply:
Sure. I'll revert that :+1: